examples/cors/apicast-cors.yml

```diff
@@ -0,0 +1,231 @@
+apiVersion: v1
+kind: Template
```

Comment:
I'd leave the config map + just shell commands to oc patch for mapping the path.
That way it should be pretty stable. 

The problem with copy of the template is that it is hard to know what changed and keep it in sync. Same rules as to code apply.